<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="2" passed="1">
  <reporter-output>
    <line>
      <![CDATA[Test Case 028 : On Clicking Ajax loader page link, link is redirecting to Ajax Loader Page]]>
    </line>
  </reporter-output>
  <suite name="MyFirstProject" duration-ms="16457" started-at="2022-06-22T14:32:02Z" finished-at="2022-06-22T14:32:19Z">
    <groups>
    </groups>
    <test name="MyFirstProject" duration-ms="16457" started-at="2022-06-22T14:32:02Z" finished-at="2022-06-22T14:32:19Z">
      <class name="com.webdriveruniversity.testcases.Ajax_Loader_Page_Test">
        <test-method status="PASS" signature="Browser_Launch(java.lang.String)[pri:0, instance:com.webdriveruniversity.testcases.Ajax_Loader_Page_Test@3ffc5af1]" name="Browser_Launch" is-config="true" duration-ms="8828" started-at="2022-06-22T14:32:02Z" finished-at="2022-06-22T14:32:11Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Browser_Launch -->
        <test-method status="PASS" signature="Test_Case_028()[pri:0, instance:com.webdriveruniversity.testcases.Ajax_Loader_Page_Test@3ffc5af1]" name="Test_Case_028" duration-ms="1226" started-at="2022-06-22T14:32:11Z" finished-at="2022-06-22T14:32:12Z">
          <reporter-output>
            <line>
              <![CDATA[Test Case 028 : On Clicking Ajax loader page link, link is redirecting to Ajax Loader Page]]>
            </line>
          </reporter-output>
        </test-method> <!-- Test_Case_028 -->
        <test-method status="FAIL" signature="Test_Case_029()[pri:0, instance:com.webdriveruniversity.testcases.Ajax_Loader_Page_Test@3ffc5af1]" name="Test_Case_029" duration-ms="5359" started-at="2022-06-22T14:32:12Z" finished-at="2022-06-22T14:32:18Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Well Done For Waiting....!!!] but found []]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Well Done For Waiting....!!!] but found []
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.webdriveruniversity.testcases.Ajax_Loader_Page_Test.Test_Case_029(Ajax_Loader_Page_Test.java:34)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Test_Case_029 -->
        <test-method status="PASS" signature="Browser_Close()[pri:0, instance:com.webdriveruniversity.testcases.Ajax_Loader_Page_Test@3ffc5af1]" name="Browser_Close" is-config="true" duration-ms="1007" started-at="2022-06-22T14:32:18Z" finished-at="2022-06-22T14:32:19Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Browser_Close -->
      </class> <!-- com.webdriveruniversity.testcases.Ajax_Loader_Page_Test -->
    </test> <!-- MyFirstProject -->
  </suite> <!-- MyFirstProject -->
</testng-results>
